core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -81,9 +77,20 @@ public void testPublishWaitForResponses() throws Exception {
     }
 
     @Test
+    public void testPublishWaitForResponsesAck() throws Exception {
+        RequesterImpl<RestPayload> requester = initRequesterForResponsesWith(1, 1000, 800, arg ->  fail());
```

Comment:
one is testing the initial configuration waitForResponses ##is 0 and we waitForAcksTimeout and this one that we had dynamicly changed status(last message was received) from 1 to 0 and waitForResponses ##become 0:smile: